packages/waku/src/lib/rsc/html-renderer.ts

```diff
@@ -48,16 +48,14 @@ const getViteServer = async () => {
   const { nonjsResolvePlugin } = await import(
     '../plugins/vite-plugin-nonjs-resolve.js'
   );
-  const viteServer = await createViteServer(
-    await mergeUserViteConfig({
-      plugins: [nonjsResolvePlugin()],
-      ssr: {
-        external: ['waku'],
-      },
-      appType: 'custom',
-      server: { middlewareMode: true, hmr: { server: dummyServer } },
-    }),
-  );
+  const viteServer = await createViteServer({
```

Comment:
Oh, I forgot this code runs on build, even though it doesn't on PRD. 😓

Reply:
haha, i just thought I forgot to put it so I put it here, but it was the reason for failure.